lib/shared/utils/led_utils.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+/// LEDウィジェットのユーティリティクラス
+class LedUtils {
+  /// 色名をColorに変換
+  static Color colorNameToColor(String colorName) {
```

Comment:
`String` 型でハードコーディングしていると，今後色を追加・変更する時少し大変かなと思います．
こんな感じで `LedColor` のデータクラスを作ってあげるといい気がします．
（ `value` は通信の際に使いたいなって思って勝手に追加しました）

```dart
enum LedColor {
  clear("クリア", 0),
  red("赤", 1),
  green("緑", 2),
  blue("青", 3);

  final String label;
  final int value;

  static const int _minimumValue = 0;

  const LedColor(this.label, this.value);

  static LedColor fromValue(int value) {
    if (value < _minimumValue || value >= LedColor.values.length) {
      throw Exception('Invalid LedColor value: $value');
    }
    return LedColor.values[value];
  }

  static LedColor fromLabel(String label) {
    for (var color in LedColor.values) {
      if (color.label == label) {
        return color;
      }
    }
    throw Exception('Invalid LedColor label: $label');
  }
}
```

```suggestion
  static Color colorNameToColor(LedColor ledColor) {
```